kuma/api/v1/plus/bookmarks.py

```diff
@@ -0,0 +1,190 @@
+import functools
+
+
+from django.conf import settings
+from django.http import (
+    HttpResponseBadRequest,
+    HttpResponseForbidden,
+    JsonResponse,
+)
+from django.middleware.csrf import get_token
+from django.utils import timezone
+from django.views.decorators.cache import never_cache
+from django.views.decorators.http import require_http_methods
+
+from kuma.bookmarks.models import Bookmark
+from kuma.documenturls.models import DocumentURL, download_url
+
+
+class NotOKDocumentURLError(Exception):
+    """When the document URL doesn't resolve as a 200 OK"""
+
+
+def is_subscriber(func):
+    @functools.wraps(func)
+    def inner(request, *args, **kwargs):
+        user = request.user
+        if not user.is_authenticated:
+            return HttpResponseForbidden("not signed in")
+        # TEMPORARY until all things auth + subscription come together.
+        if not settings.FAKE_USER_SUBSCRIBER_NUMBER:
+            return HttpResponseForbidden("not a subscriber")
+        # Note: Deliberately commented out until we have figured out OIDC!
+        # if not UserSubscription.objects.filter(
+        #     user=user, canceled__isnull=True
+        # ).exists():
+        #     return HttpResponseForbidden("not a subscriber")
+        return func(request, *args, **kwargs)
+
+    return inner
+
+
+@never_cache
+@require_http_methods(["GET"])
+@is_subscriber
+def bookmarks(request):
+    try:
+        page = int(request.GET.get("page") or "1")
```

Comment:
I prefer the `or` because if the the URL is `?page=` what `request.GET.get('page')` is then is `''` (empty string). 
In the this context, it can't be None but imagine it was:
`request.GET['page'] = None`
you'd ultimately end up doing `int(None)`
But using `or` will ignore the falsy things like `''` or `None`.

Reply:
@peterbe Ah, good point! 👍